doc/remoteExams.md

```diff
@@ -0,0 +1,97 @@
+
+# Remote exam configurations
+
+This page lists sample assessment configurations for remote exams, where students are not physically present in the same location as the proctors.
+
+
+## Synchronous, timed exams
+
+This configuration is good when:
+
+* Almost all students take the exam at the same time
+* Some students have accommodations, such as 1.5x time -- **these students must be listed first in the access rules**
```

Comment:
```suggestion
* Some students have accommodations, such as 1.5x time &mdash; **these students must be listed first in the access rules**
```

you should use the emdash &mdash; it's extra fancy :slightly_smiling_face:

Reply:
true, and I even know how to type them — I think I got lazy from the MacOS slack client which auto-converts double-dashes to em-dashes 😄